src/app/components/v2/modal-share/modal-share.js

```diff
@@ -10,20 +10,22 @@ export const reduxConfig = { actions, reducers, initialState };
 
 const mapStateToProps = ({ modalShare, location }) => ({
   isOpen: modalShare.isOpen,
-  currentLocation: window.location.href,
+  currentLocation: `https://www.half-earthproject.org/maps?${  window.location.href.split('?')[1]}`,
```

Comment:
I use the same string in three places here... but I don't know how to instantiate a variable for this :( or a constant for that matter.

Reply:
```js
const mapStateToProps = ({ modalShare, location }) => {
  const currentLocation = `https://www.half-earthproject.org/maps${location.search}`;
  return ({
    isOpen: modalShare.isOpen,
    currentLocation,
    urlToCopy: modalShare.urlToCopy,
    linkActive: modalShare.linkActive,
    coordinates: location && location.query && location.query.coordinates,
    orientation: location && location.query && location.query.orientation,
    shareSocialMedia: [
      {
        icon: facebookIcon,
        link: `https://www.facebook.com/sharer/sharer.php?u=${currentLocation}`,
        className: 'facebookIcon'
      },
      {
        icon: twitterIcon,
        link: `https://twitter.com/intent/tweet?url=${currentLocation}`,
        className: 'twitterIcon'
      }
    ]
  });
};
```

Reply:
Thanks guys! 🙇  much appreciated. I'll make the change.